### ISSUE TITLE
AppBundle publication error: The file is being processed. It may take 2-5 minutes, depending on the size of the software package

### ISSUE THREAD

**Issue by cosic** [cianru/huawei-publish-gradle-plugin#7]
Reproduce on library v1.1.0 with options 
```
{
   ...
   publish = true
   buildFormat = "aab"
}
```

Output:
```
> Task :app:publishHuaweiAppGalleryRelease
Huawei AppGallery Publishing API: Found build file: `app-release.aab`
Huawei AppGallery Publishing API: Get Credentials
Huawei AppGallery Publishing API: Get Access Token
Huawei AppGallery Publishing API: Get App ID
Huawei AppGallery Publishing API: Get Upload Url
Huawei AppGallery Publishing API: Upload build file
Huawei AppGallery Publishing API: Update App File Info
Huawei AppGallery Publishing API: Submit Release
> Task :app:publishHuaweiAppGalleryRelease FAILED
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':app:publishHuaweiAppGalleryRelease'.
> Ret(msg=[cds]submit failed, additional msg is [The file is being processed. It may take 2-5 minutes, depending on the size of the software package.])
```

**Comment by minamak91** [comment]
@cosic Is the commit attached should fix the issue? If yes, could you please merge the develop to the master?

**Comment by cosic** [comment]
@minamak91 Hi! I've published snapshot_1.2.0. Could you check AppBundle publishing with this version without release phase? The release phase doesn't work yet due to issue at Huawei Rest API. For more information see READNE at [snapshot-1.2.0](https://github.com/cianru/huawei-publish-gradle-plugin/tree/snapshot-1.2.0) branch

**Comment by minamak91** [comment]
@cosic This doesn't work for me and when I tried to publish the branch on Jitpack, it fails...
```
classpath "ru.cian:huawei-publish-gradle-plugin:1.2.0-SNAPSHOT"
```
Could you please just tell me what should be the version added to the build gradle classpath?
Thanks in advance :)

**Comment by cosic** [comment]
@minamak91 To use snapshot you must add sonatype snapshot repository: 
```
buildscript {
    repositories {
        maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
    }

    dependencies {
        classpath "ru.cian:huawei-publish-gradle-plugin:1.2.0-SNAPSHOT"
    }
}
```

**Comment by minamak91** [comment]
@cosic It works fine the publishing without release phase, thank you so much :+1:

**Comment by cosic** [comment]
Released at `v1.2.1`
